### ISSUE TITLE
Discussion: Accessbility testing in dialogporten-frontend

### ISSUE THREAD

**Issue by alexdigdir** [Altinn/dialogporten-frontend#1592]
**Background**

Improve accessibility *integration* test coverage.

This is a task about exploring the possibilities for expanding existing (some?) playwright tests to uncover accessibility violations. **Maybe better:** Alternatively separate tests for accessibility tests with Playwright can be written and probably the cleanest. 

There is only one test now, `axe.test.ts`, which scans for accessibility violations which is executed with `pnpm test:accessibility` (which runs `... playwright test --project=accessibility`)

All 91 violations are being ignored, and this step is not part of the CI/CD.

Tasks:

- Explore best continuation of writing accessibility tests with playwrights (cf. above)
- All tests need to be part of the `accessibility` project in order to run with 
- Ensure correct wcag rules (2.1 aa-aaa) are set for axe when scanning
- Change `pnpm dlx` instead of `npx` for Playwright scripts in `package.json` for `frontend` project
- Evaluate violations and try to address their source

**Outcome**

How do we go from there? What should be be covered by component library dependencies and what should be part of the test suites for `dialogporten-frontend`?

Update: 
Accessibility issues in Altinn components are resolved, however we are still migrating components in arbeidsflate.
To avoid double work, these tests should be enabled (and fixed if any issues found), after migrating job is finished.

```[tasklist]
### Tasks
- [x] Chore: Change pnpm dlx instead of npx for Playwright scripts in package.json for frontend project (needed?)
- [x] Chore: Ensure correct wcag rules (2.1 aa-aaa) are set for axe when scanning (test - frontend)
- [x] Add AxeBuilder analysis for each route in App
- [x] Only inspect violations with impact "serious" and "critical" and add to CI/CD
- [x] Chore: Setup accessibility tests for altinn-components and resolve issues
```
